chapter_recurrent-modern/gru_vn.md

```diff
@@ -275,7 +276,11 @@ We instantiate all weights and biases relating to the update gate, the reset gat
 Subsequently, we attach gradients to all the parameters.
 -->
 
-*dịch đoạn phía trên*
+Bước tiếp theo là để khởi tạo các tham số mô hình.
+Chúng ta khởi tạo giá trị của các trọng số từ phân phối Gauss với phương sai là $0.01$ và thiết lập các hệ số điều chỉnh bằng $0$.
```

Comment:
```suggestion
Chúng ta khởi tạo giá trị của các trọng số từ phân phối Gauss với phương sai là $0,01$ và thiết lập các hệ số điều chỉnh bằng $0$.
```

Reply:
Số dùng dấu `.` em ơi, xem lại text gốc :D

Reply:
Tiếng Việt số thực mình dùng dấu `,` nên nhóm mình muốn theo cách ghi của tiếng Việt á anh

Reply:
chỗ này đâu có số nhiều cho "bias" nhỉ?

Reply:
Em xin phép được mở lại conversation này. @cuongvng có nói là nếu mình chuyển `.` thành `,` thì nó sẽ không thống nhất với code. Theo ý kiến của mình thì văn bản và code có thể coi như 2 ngôn ngữ khác nhau nên không ảnh hưởng nhiều. Từ lúc dịch cuốn MYL thì mọi người có thống nhất sẽ ghi theo cách của tiếng Việt. Tuy nhiên nếu thấy không ổn thì mình có thể discuss lại nhé

Reply:
Nếu vậy mình nghĩ ít nhất nên lưu ý cho độc giả ở phần [ký hiệu](https://d2l.aivivn.com/chapter_notation/index_vn.html) rằng decimal point là dấu phẩy theo quy ước trong tiếng Việt. Mình vẫn nghiêng về kí hiệu decimal point là `.`, thứ nhất là để nhất quán, hai là khi dịch và review cũng thường khó để ý hết chỗ này. Ví dụ:
![image](https://user-images.githubusercontent.com/30251584/83526993-640fb700-a511-11ea-90ff-b1a860387990.png)
(Fun) ko biết mọi người thế nào chứ khi viết bằng giấy bút mình dùng `,`, chắc quen từ nhỏ, còn typing thì toàn dùng `.` 😄

Reply:
Good point! Mình có thể sẽ cần một cuộc "tổng kiểm duyệt" để rà lại cho đúng theo thống nhất. Anh @duythanhvn thấy sao ạ?

P/s: Mình toàn dùng `.` cả khi typing lẫn khi viết 😆

Reply:
Ơ, anh tưởng chốt dùng `.` hồi đợt em thảo luận rồi chứ @rootonchair . Duyệt nhé, anh sẽ kiểm tra, bổ sung trong thời gian tới hen.

Reply:
Summarize lại

* Số thực: dùng `,`
* Phân cách các giá trị trong số nguyên: dùng `.`

Reply:
> Ơ, anh tưởng chốt dùng `.` hồi đợt em thảo luận rồi chứ @rootonchair . Duyệt nhé, anh sẽ kiểm tra, bổ sung trong thời gian tới hen.

`.` là dùng cho phân cách số nguyên anh ơi. Cái này bọn em đang nói dùng `,` cho decimal point

Reply:
> Summarize lại
> 
> * Số thực: dùng `,`
> * Phân cách các giá trị trong số nguyên: dùng `.`

Quy ước này bị ngược à em?
Anh thấy quy ước ngược lại là quy ước chuẩn (tiếng Việt hay tiếng Anh).

Reply:
> > Summarize lại
> > 
> > * Số thực: dùng `,`
> > * Phân cách các giá trị trong số nguyên: dùng `.`
> 
> Quy ước này bị ngược à em?
> Anh thấy quy ước ngược lại là quy ước chuẩn (tiếng Việt hay tiếng Anh).

Tiếng Việt ạ. Ví dụ `0,1`, `0,25` thay vì `0.1`, `0.25`

Reply:
> chỗ này đâu có số nhiều cho "bias" nhỉ?

Tác giả viết không thống nhất lắm anh ạ, trên thì số ít dưới số nhiều @@

Reply:
Cách thứ nhất (được sử dụng phổ biến hơn)
> Sau chữ số hàng nghìn, triệu, tỷ, nghìn tỷ, triệu tỷ, tỷ tỷ phải đặt dấu chấm (.), nếu có ghi chữ số sau chữ số hàng đơn vị phải đặt dấu phẩy (,) sau chữ số hàng đơn vị

là cách em đang nói ở trên đó anh

Reply:
Nhưng mà sau khi anh đọc qua các cuốn sách giáo khoa toán ở VN thì chúng ta dùng dấu `.` để ngăn cách với phần thập phân :-?

Reply:
Tạm thời để tránh blocking thì mình sẽ giữ nguyên nhé. Em sẽ có một thread để bàn luận về vấn đề này.

P/s: anh đừng ẩn thread này để em dễ tìm lại sau này